airbyte-integrations/connectors/source-k6-cloud/source_k6_cloud/schemas/tests.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "k6-runs": {
+      "type": "array"
+    },
+    "k6-runs-trending-metrics": {
+      "type": "array"
+    },
+    "k6-tests": {
+      "type": "array"
+    },
+    "load_zones": {
+      "type": "array"
+    },
+    "meta": {
+      "type": "object"
```

Comment:
This schema is not correct. I think users are more interested in the data inside the `k6-tests`. Even if you want to retrive all these please add the schema for k6-tests.

Reply:
Fixed